src/App.jsx

```diff
@@ -14,45 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_APHABETICALY = 'alphabetical';
+const SORT_BY_LENGTH = 'length';
+
+function sortGoods(goods, sortMethod, isReversed) {
+  let sortedGoods = [...goods];
+
+  switch (sortMethod) {
+    case SORT_APHABETICALY:
+      sortedGoods = sortedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+
+    case SORT_BY_LENGTH:
+      sortedGoods = sortedGoods.sort((a, b) => a.length - b.length);
```

Comment:
No need to set ` sortedGoods ` again (lines 27 and 31) as method `.sort()` mutates the array anyway (just like `.reverse()`) which also means you can define ` sortedGoods ` with ` const `. (I had a similar minor mistake 😉) ![1x](https://github.com/mate-academy/react_list-of-goods-js/assets/146858277/e1529eb7-c3b1-4a44-b9a7-89b5e11bff2c)